src/test/apiTest/resources/application.yaml

```diff
@@ -133,4 +133,7 @@ hmcts_service_id: BHA1
 launchdarkly:
   sdk-key: ${LAUNCH_DARKLY_SDK_KEY:sdk-key}
   offline-mode: false
-  env: ${LAUNCH_DARKLY_ENV:default}
\ No newline at end of file
+  env: ${LAUNCH_DARKLY_ENV:default}
+
+bundlesClaimantSubmittedRespondentNotificationTemplateId: ${BUNDLES_CLAIMANT_SUBMITTED_RESPONDENT_NOTIFICATION_TEMPLATE_ID:5c829358-0b54-4335-9d60-1c8296c4a516}
```

Comment:
Same as before. Move inside template block

Reply:
The variable is moved within the template block